site/src/pages/[shortname]/[blockSlug].page.tsx

```diff
@@ -164,6 +164,30 @@ const BlockPage: NextPage<BlockPageProps> = ({
     return catalog.filter(({ name }) => name !== blockMetadata.name);
   }, [catalog, blockMetadata]);
 
+  const repositoryUrl: URL | null = useMemo(() => {
+    if (typeof blockMetadata.repository === "string") {
+      return new URL(blockMetadata.repository);
+    }
```

Comment:
This does not appear to correctly handle the [shorthand syntax](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository)